lib/happo/public/HappoDiffs.jsx

```diff
@@ -133,7 +133,7 @@ function SelectedView({ image, selectedView }) {
 }
 SelectedView.propTypes = {
   image: PropTypes.shape(imageShape).isRequired,
-  selectedView: PropTypes.string.isRequired,
+  selectedView: PropTypes.oneOf(Object.keys(VIEWS).map(key => VIEWS[key])).isRequired,
```

Comment:
we do this in two places now, so it might be worth initializing it at the top somewhere.